tools/localise-html.py

```diff
@@ -47,7 +47,7 @@ def handle_starttag(self, tag, attrs):
             """This is where html's dir attribute is set to ltr or rtl"""
             if tag == "html":
                 text = self.get_starttag_text()
-                self.p('<html dir="%s" lang=en' % ('rtl' if self.localename in rtl_languages else 'ltr') + text[text.index('html') + 4:])
+                self.p('<html dir="%s" lang=' % ('rtl' if self.localename in rtl_languages else 'ltr') + self.alpha2name + text[text.index('html') + 4:])
```

Comment:
It seems like this would be far better as `'<html dir="%s" lang="%s"'`? Your current version is also missing the quotes (it's valid but not great HTML).